config/default/kustomization.yaml

```diff
@@ -20,6 +20,7 @@ resources:
 - ../manager
 - ../webhook
 - ../busola
+- ../ui-extensions
```

Comment:
we have two folders containing configuration for busola:
- busola 
- ui-extensions

Please consolidate into one

Reply:
KEDA uses `ui-extensions`, Telemetry uses `busola`. We may use `ui-extensions` as it seembs better structured, we need to convert exising `busola` in EM. NATS manager already has `ui-extensions`.